mex/model/entities/distribution.json

```diff
@@ -22,96 +24,69 @@
                 }
             ],
             "default": null,
+            "description": "A data service that gives access to the distribution of the dataset ([DCAT, 2020-02-04](https://www.w3.org/TR/2020/REC-vocab-dcat-2-20200204/)).",
             "sameAs": [
                 "http://www.w3.org/ns/dcat#accessService"
             ]
         },
         "accessURL": {
-            "anyOf": [
-                {
-                    "$ref": "/schema/fields/link"
-                },
-                {
-                    "type": "null"
-                }
-            ],
-            "default": null,
-            "subPropertyOf": [
-                "http://www.w3.org/ns/dcat#accessURL"
-            ]
-        },
-        "author": {
             "default": [],
+            "description": "A URL of the resource that gives access to a distribution of the dataset. E.g. landing page, feed, SPARQL endpoint ([DCAT, 2020-02-04](https://www.w3.org/TR/2020/REC-vocab-dcat-2-20200204/)).",
             "items": {
-                "$ref": "/schema/entities/person#/identifier"
+                "anyOf": [
```

Comment:
inside of lists, we don't want to support `null` values. so this `anyOf` should be scratched, we just need `items: {ref: link}`.

same goes `downloadURL`